src/execute_system.c

```diff
@@ -77,6 +77,7 @@ int	system_exec(t_command *cmd)
 	int		status;
 
 	pid = fork();
+	signal(SIGINT, child_signal_handler);
```

Comment:
What's the better way to test this signal? I've tested in many ways and all worked.

Reply:
I don't know. What is this supposed to do?

Reply:
Handle signals coming from child proccess

Reply:
Maybe we should test it separate as a main proccess and see how it handles signals